keepercommander/importer/keepass/keepass.py

```diff
@@ -186,7 +175,21 @@ def do_import(self, filename):
                                 elif key == 'Notes':
                                     record.notes = value
                                 else:
-                                    record.custom_fields[key] = value
+                                    if key[0] == '$':
+                                        pos = key.find(':')
```

Comment:
A lot of uses of str.find can be simplified with, EG:
>>> s = 'abc:def:ghi'
>>> s.partition(':')
('abc', ':', 'def:ghi')

There's also an rpartition method for starting from the other end of the string.

Reply:
Yes, it code better. I did not know about `partition`